sigar_windows_test.go

```diff
@@ -101,25 +99,21 @@ var _ = Describe("SigarWindows", func() {
 			connList := sigar.NetTcpConnList{}
 			err := connList.Get()
 			Ω(err).ShouldNot(HaveOccurred())
-			fmt.Printf("Connections: %v\n", connList)
 		})
 		It("gets UDP IPv4 conns", func() {
 			connList := sigar.NetUdpConnList{}
 			err := connList.Get()
 			Ω(err).ShouldNot(HaveOccurred())
-			fmt.Printf("Connections: %v\n", connList)
 		})
 		It("gets TCP IPv6 conns", func() {
 			connList := sigar.NetTcpV6ConnList{}
 			err := connList.Get()
 			Ω(err).ShouldNot(HaveOccurred())
-			fmt.Printf("Connections: %v\n", connList)
 		})
 		It("gets UDP IPv6 conns", func() {
 			connList := sigar.NetUdpV6ConnList{}
 			err := connList.Get()
 			Ω(err).ShouldNot(HaveOccurred())
```

Comment:
Does `gomega` really use the capital omega for it's matchers?  I guess that must be how everyone else in the world feels about code using the latin alphabet :P